src/config/networks/network.d.ts

```diff
@@ -0,0 +1,59 @@
+import { ETHEREUM_NETWORK } from 'src/logic/wallets/getWeb3'
```

Comment:
can we do the opposite? I mean defining the networks here and import them in `'src/logic/wallets/getWeb3'`

Reply:
I agree, but we preferred doing it later (in a different PR) because there are lots of files that import this enum, and it will make a lot of noise.

And probably we should wrap the `ETHEREUM_NETWORK[getNetwork()]` into a function.

Reply:
Don't forget to create a ticket for this

Reply:
@dasanra, @nicosampler it's done in
[`4e5e8bf` (#1422)](https://github.com/gnosis/safe-react/pull/1422/commits/4e5e8bf6ba5d744e62beb7935d22caaec6395d0b)